b2/console_tool.py

```diff
@@ -58,6 +58,10 @@
 # Disable for 1.* behavior.
 VERSION_0_COMPATIBILITY = False
 
+# Optional Env variable to use for getting account info while authorizing
+B2_ACCOUNT_ID_ENV_VAR = 'B2_ACCOUNT_ID'
```

Comment:
```suggestion
B2_APPLICATION_KEY_ID_ENV_VAR = 'B2_APPLICATION_KEY_ID'
```
nomenclature was changed, that's the new correct name